src/main/java/dev/blachut/svelte/lang/codeInsight/SvelteAnnotator.kt

```diff
@@ -0,0 +1,20 @@
+package dev.blachut.svelte.lang.codeInsight
+
+import com.intellij.lang.annotation.AnnotationHolder
+import com.intellij.lang.annotation.Annotator
+import com.intellij.lang.javascript.highlighting.JSHighlighter
+import com.intellij.lang.javascript.psi.JSLabeledStatement
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiElement
+import dev.blachut.svelte.lang.SvelteFileViewProvider
+
+class SvelteAnnotator : Annotator {
```

Comment:
This annotator is buggy, doesn't highlight the label but random things after it.

![image](https://user-images.githubusercontent.com/615013/62624649-d7619000-b923-11e9-942f-d55e26cf7412.png)
![image](https://user-images.githubusercontent.com/615013/62624667-e0eaf800-b923-11e9-8c61-e54645879b92.png)
![image](https://user-images.githubusercontent.com/615013/62624685-e9dbc980-b923-11e9-97bd-f06b68666bbc.png)